### ISSUE TITLE
Documentation Example leads to error

### ISSUE THREAD

**Issue by s2t2** [aschleg/petpy#9]
Hi, I'm trying to use the examples you've provided (version `1.8.2 `), but I'm running into an error. I'm hoping you might be able to provide some context about what might be going on, and how to fix it.

My code:

```py
import petpy

API_KEY = "abc123" # for example

pf = petpy.Petfinder(API_KEY)
print(type(pf)) #> #> <class 'petpy.api.Petfinder'>

cats_list = pf.breed_list("cat") #> json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
print(type(cats_list))


```

I was using Python 3.7 but then I saw your package might not support that version yet, so I tried Python 3.6 which the docs say should be supported, but the issue remains. 

Please advise!

**Comment by s2t2** [comment]
UPDATE: I tried `cats_list = pf.breed_list("cat", return_df=True)` but that throws the same error.

**Comment by s2t2** [comment]
Also tried rolling back to the following versions of the package (using Python 3.6) but no luck:

  + 1.8.0
  + 1.7.2
  + 1.7.0

Assuming the package works, what am I missing?

**Comment by s2t2** [comment]
I read that the API recently changed from version 1 to version 2 and maybe that's breaking things. So I tried passing in the API secret value, but that didn't work either:

```py
pf = petpy.Petfinder(API_KEY, secret=CLIENT_SECRET)
pets_list = pf.pet_find(location="Seattle", animal="cat", sex="Female", count=100, return_df=True)
```

I see there might be a `host` value, but it is hard coded. I'm going to try to fork the repo and update this initializer to accept a `host` param, or update it to default to the new URL: https://api.petfinder.com/v2/

**Comment by s2t2** [comment]
Hmm, using the new host doesn't work. (see #10) Going to have to call it quits for the night, but hoping to hear from you soon. 😸

**Comment by aschleg** [comment]
Hi Mike,

Thank you so much for investigating this and apologies for the delay in my response! I haven't had a chance to dig into this yet, but my first assumption is perhaps something changed on the PetFinder API side that's causing the JSON parsing issue.

**Comment by aschleg** [comment]
Hello!

Apologies again for the delay and thanks again for bringing this to my attention. It turns out the issue was much larger than anticipated as Petfinder did a huge rework to their API so it invalidated a lot of the methods and code that the package originally had. 

I just released version 2.0.0 of petpy which updates the package to work with v2.0 of the Petfinder API. Due to this, you'll have to change some of your code above to the new methods in the new version of `petpy`. Petfinder also now requires the secret key you received from them to authenticate the connection as well. 

The updated [README](https://github.com/aschleg/petpy) should have all the basic stuff and the [docs](https://petpy.readthedocs.io/en/latest/) are also updated to be in line with the new version of the API and petpy.

Happy to help if you have any questions! I'll leave this issue open for now in case you run into any issues.

**Comment by aschleg** [comment]
Going to close this issue for now but please let me know if you encounter any difficulties!
